Automattic/jetpack
# Boost: Fix performance history tooltip

Fixes #39827

## Proposed changes:
* Fix the tooltip in performance history

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:
* Visit Boost dashboard using a premium site.
* Check to make sure the day tooltips appear and behave correctly
